src/parser/parseSVGDocument.ts

```diff
@@ -1,7 +1,6 @@
 //@ts-nocheck
 
 import { fabric } from '../../HEADER';
-import { toArray } from '../util';
```

Comment:
why not import directly from the file `lang_array` and not from the index?
That will solve the circular dep

Reply:
Is what i was doing i removed for now all imports from the main index in util, and where possible i m linking to direct files.
Is probably better to leave the util/index.ts  just to create the external facing export for who will use fabric.util